apps/frontend/src/__tests__/types/global.d.ts

```
@@ -0,0 +1,23 @@
+import type { hasPermissionFor } from "~/lib/utils";
+import type { jest } from "@jest/globals";
+type AuthFn = typeof auth;
+type HasPermissionForFn = typeof hasPermissionFor;
+
+declare global {
+  var mockPrisma: DeepMockProxy<PrismaClient>;
+  var mockAuth: jest.MockedFunction<() => Promise<Session | null>>;
+  var mockHasPermissionFor: jest.MockedFunction<HasPermissionForFn>;
+
+  // Jest global
+  var jest: typeof jest;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'jest'. Consider to delete it or rename it.

'jest' is defined here:


(lint/suspicious/noRedeclare)